components/desktop/gnome2/planner/Makefile

```diff
@@ -25,11 +26,11 @@ COMPONENT_ARCHIVE_HASH=	\
     sha256:130d29e6d6b8c1994017deba2acca05a4bf83ba1e169635ec8c5c7878abe0696
 COMPONENT_ARCHIVE_URL=	http://ftp.gnome.org/pub/GNOME/sources/$(COMPONENT_NAME)/0.14/$(COMPONENT_ARCHIVE)
 
-include $(WS_TOP)/make-rules/prep.mk
-include $(WS_TOP)/make-rules/configure.mk
-include $(WS_TOP)/make-rules/ips.mk
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
 
-PATH=/usr/gnu/bin:/usr/bin:/usr/perl5/bin
+PATH = $(PATH.gnu)
 
 COMPONENT_PREP_ACTION =        ( cd $(@D) && \
 					aclocal -I. && \
```

Comment:
The gnome-media component fails with messages from pkgdepend like:

Couldn't find 'usr/bin/gnome-volume-control-applet' in any of the specified search directories:

Similarly, the libcanberra component fails with these errors:

Couldn't find 'usr/lib/libcanberra-gtk3.so.0.1.7' in any of the specified search directories:
Couldn't find 'usr/lib/pkgconfig/libcanberra-gtk3.pc' in any of the specified search directories:

The libchamplain component fails with a different error:

.../components/library/libchamplain/build/manifest-sparc-libchamplain.depend has unresolved dependency '
    depend type=require fmri=__TBD \
        pkg.debug.depend.file=libclutter-gtk-0.10.so.0 \
        pkg.debug.depend.reason=usr/lib/libchamplain-gtk-0.8.so.1.1.4 \
        pkg.debug.depend.type=elf \
        pkg.debug.depend.path=lib \
        pkg.debug.depend.path=usr/lib'.

I don't know what to do about these errors.